server/index.js

```diff
@@ -2,20 +2,106 @@ const express = require('express');
 const bodyParser = require('body-parser');
 
 const {mainProduct} = require('../database/mainProduct.js');
+const {photos} = require('../database/mainProduct.js');
 
 const app = express();
 const PORT = 3000;
 
+const s3 = require('./s3_photoViewer')
+const presignedGETURL = require('./s3_photoViewer')
+
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({extended: true}));
 
-//UNCOMMENT FOR REACT
 app.use(express.static(__dirname + '/../client/dist'));
 
-// UNCOMMENT FOR ANGULAR
-// app.use(express.static(__dirname + '/../angular-client'));
-// app.use(express.static(__dirname + '/../node_modules'));
+// Create params for S3.deleteBucket
+var bucketParams = {
+  Bucket : 'amazon-main-product-bucket'
+};
+
+// UPDATE S3 images to db
+app.put('/api/photos', function(req, res) {
```

Comment:
this is unnecessary, the urls should be added there in the seeding script. If there are random numbers you would maybe need to pull from amazon to get the appropriate number of urls but it seems like yours are standard so this shouldn't be an issue.